src/common/featuremanager/FeatureManagerService.js

```diff
@@ -956,6 +956,18 @@
       }
       return deferredResponse.promise;
     };
+
+    this.coordinatesWithinRange = function(coords1, coords2, range) {
+      if (coords1.length != coords2.length) {
+        return false;
+      }
+      for (var i = 0; i < coords1.length; i++) {
```

Comment:
@travislbrundage  - This is the only real change I'd suggest at this point, everything else looks good!